README.md

```diff
@@ -1,5 +1,40 @@
-# Northcoders House of Games API
+# Aidan's Games API
 
-## Creating environment variables
+## Introduction
+---
+This is an API centered around board games. It has the functionality to upvote board game reviews, post comments on reviews as well as much more. For more information about each endpoint and their functionality use the "/api" endpoint on the hosted version. This is a link to the heroku hosted version: https://nc-games-aidan.herokuapp.com/
 
-To connect to the two databases locally a package called .env can be used. Firstly, you must create two new files called `.env.test` and `.env.development`, these files will store our variables. To the `.env.test` file add `PGDATABASE=<test-database-name>` and then to the `.env.development` add `PGDATABASE=<dev-database-name>`. The names of each database can be found in `/db/setup.sql` incase you are curious.
\ No newline at end of file
+The minimum version of node that must be installed is v17.7.1 and the minimum version of Postgres that must be installed is v12.10.
+<br>
+
+## Setup
+---
+### Cloning
+Copy the url of the github repo. In this case the url is https://github.com/Oathian/NC-Games-Backend
+Open the terminal and navigate to where you would like the file to be located. Then type: 
+```
+git clone https://github.com/Oathian/NC-Games-Backend
+```
+<br>
+
+### Installing dependencies
+
+Next, to install any dependencies type `npm install` or `npm i` in your terminal.
+
```

Comment:
It might be worth making it a little clearer that you need to navigate into the repo before running this